pkg/security/scc/byrestrictions.go

```diff
@@ -54,7 +83,7 @@ func pointValue(constraint *securityapi.SecurityContextConstraints) int {
 // than Secret, ConfigMap, EmptyDir, DownwardAPI, Projected, and None will result in
 // a score of 5.  If the SCC only allows these trivial types, it will have a
 // score of 0.
-func volumePointValue(scc *securityapi.SecurityContextConstraints) int {
+func volumePointValue(scc *securityapi.SecurityContextConstraints) weight {
```

Comment:
Let's call it one thing if we're going to call it anything.  Then let's get @adelton's PR unblocked :smile: